### PR TITLE
feat(workflow): List registered queries in error response when a quer…

### PR DESCRIPTION
**Pull request by bergundy**
…y is not found

This is required by the UI to display available queries.
Eventually we should implement a `__get_handlers` internal query but this will do for now.

### CODE REVIEW

**Review by lorensr — APPROVED** [review]
test?

**Comment by bergundy** [comment]
I was lazy at this late hour, I tested it manually.
Yes, this deserves a test.

File: packages/test/src/integration-tests.ts [hunk 1]
```diff
@@ -391,6 +392,21 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
...
+    const workflow = await client.start(workflows.unblockOrCancel, {
+      taskQueue: 'test',
+      workflowId: uuid4(),
+    });
+    await workflow.signal(workflows.unblockSignal);
+    await workflow.result();
+    await t.throwsAsync(workflow.query('not found'), {
+      instanceOf: QueryNotRegisteredError,
+      message:
+        '3 INVALID_ARGUMENT: Workflow did not register a handler for not found. Registered queries: [__stack_trace isBlocked]',
```
Comment (lorensr) [inline packages/test/src/integration-tests.ts:406]: Is this part helpful? `3 INVALID_ARGUMENT: ` Wonder if we should remove

Comment (bergundy) [reply packages/test/src/integration-tests.ts:406]: I didn't want to mess with the returned message, I guess it's okay to trim.

**Pull request merged**